src/common/DecayCounter.cc

```diff
@@ -62,7 +62,7 @@ void DecayCounter::generate_test_instances(list<DecayCounter*>& ls)
 
 void DecayCounter::decay(utime_t now, const DecayRate &rate)
 {
-  if (now >= last_decay) {
+  if (last_decay != utime_t() && now >= last_decay) {
```

Comment:
I think we should avoid zero las_decay at the source, how about updating last_decay when decoding decay counter

Reply:
Good idea! I will update it accordingly.